docs/enhancements/2023-09-01-cross-silo-error.md

```diff
@@ -0,0 +1,28 @@
+## Summary
+### General Motivation
+Before this proposal, when the execution of a DAG encounters an error in 'alice', below is what will happen:
+![image](https://github.com/ray-project/rayfed/assets/26196566/77fa1bd1-dc14-4ddc-8d9e-5dbaf6942564)
+
+In alice, both main thread and data sending thread will raise the error, and the process will exit.
+In bob,  since it needs the input from 'alice', it will waits for 'alice' forever no matter whether 'alice' exists or not.
```

Comment:
```suggestion
In bob,  since it needs the input from 'alice', it waits for 'alice' forever no matter whether 'alice' exists or not.
```
or `it will wait for`